packages/app/src/app/components/StripeMessages/UpgradeSSEToV2.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useEffects, useAppState } from 'app/overmind';
+import { MessageStripe } from '@codesandbox/components';
+import { sandboxUrl } from '@codesandbox/common/lib/utils/url-generator';
+import { hasPermission } from '@codesandbox/common/lib/utils/permission';
+
+export const UpgradeSSEToV2Stripe = () => {
+  const state = useAppState();
+  const effects = useEffects();
+  const updateSandbox = useEffects().api.updateSandbox;
+
+  return (
+    <MessageStripe variant="primary">
+      <span>
+        This sandbox runs much faster in our new Editor. Do you want to convert
+        it to a Cloud Sandbox?
+      </span>
+      <MessageStripe.Action
+        onClick={async () => {
```

Comment:
Do we want some loading state? Not sure how long it takes for the sandbox to be transformed into a v2 sandbox and the user is redirected?

Reply:
If we need it you can use the loading button:
 
<img width="270" alt="image" src="https://user-images.githubusercontent.com/1891339/213707088-b512aec5-1cd8-4ab6-9209-9421eedcde37.png">